tests/configuration_wrapper_ut/test_configuration_wrapper.h

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef TEST_CONFIGURATION_WRAPPER_H
+#define TEST_CONFIGURATION_WRAPPER_H
+
+#ifdef __cplusplus
+#include <cinttypes>
+#else
+#include <inttypes.h>
```

Comment:
We should not have these anymore. Andrei is having a push to remove them everywhere actually.